skrifa/src/outline/tricky.rs

```diff
@@ -0,0 +1,140 @@
+//! Match FreeType's "tricky" font detection.
```

Comment:
Suggest to rephrase to "Name detection for fonts that require hinting to be run for correct contours (FreeType calls these "tricky" fonts."

We could also call it something like hinting-reliant or so, if we need a single word.

Reply:
I like this comment so I used it verbatim.